SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -1008,72 +1017,127 @@ CSU2TCLib::CSU2TCLib(const CConfig* config, unsigned short val_nDim, bool viscou
     Tcf_a[20] = 1.0; Tcf_b[20] = 0.0; Tcb_a[20] = 0.5; Tcb_b[20] = 0.5;
     Tcf_a[21] = 0.0; Tcf_b[21] = 1.0; Tcb_a[21] = 0;   Tcb_b[21] = 1;    
 
-    //TODO: Implement Collision Integral Data for AIR-7 (JN)
     /*--- Collision integral data ---*/
-    //TODO: ELECTRONS HAvE NO OMEGAS....NEED TO ALTER LOOPS? or add dummy vector.....
+    // Index 1: collider
+    // Index 2: partner
+    // Index 3: A1, A2, A3
+
     // Omega(0,0) ----------------------
+    //e-  (TODO: NO+ and e-,  NO+ and NO+, e- and e- ??????)
+    Omega00(0,0,0) = 0.0000000E+00;  Omega00(0,0,1) = 0.0000000E+00;  Omega00(0,0,2) = 0.0000000E+00;  Omega00(0,0,3) = 0.0000000E+00;
```

Comment:
@jtneedels these values are undefined in Scalabrin's thesis.  Something to look into

Reply:
I believe these are detailed in Table A.10 where attractive/repulsive Coulomb potential is discussed. Since these are all ion-ion pairs, I think the collision cross section is based on Coulomb potential since all are either attractive/repulsive.